src/main/target/CC3D/target.h

```diff
@@ -44,7 +44,7 @@
 
 #define USABLE_TIMER_CHANNEL_COUNT 12
 
-#define DEBUG_MPU_DATA_READY_INTERRUPT
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
```

Comment:
it was enabled once the code was added for CC3D so people could test it on CI builds, it never got commented out later.